core/src/main/java/org/jdbi/v3/mapper/CollectorFactory.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jdbi.v3.tweak;
+package org.jdbi.v3.mapper;
```

Comment:
I'm torn here.  We're getting a lot of packages, and Collectors are intimately tied to mapping and only have a few classes...